tuf/fetcher.py

```diff
@@ -16,6 +16,16 @@
 """
 
 import abc
+import requests
+import six
+import logging
+import time
+
+import urllib3.exceptions
+import tuf.exceptions
+import tuf.settings
```

Comment:
It is not compatible but I figured this will land before the imports (and since merge fixes are going to be needed anyway I'd change these in the imports PR where I change everything else as well)